lib/test_prof/factory_prof/factory_builders/fabrication.rb

```diff
@@ -10,7 +10,7 @@ module FactoryBuilders
       class Fabrication
         # Monkey-patch Fabrication
         def self.patch
-          TestProf.require 'fabrication', ""
+          TestProf.require 'fabrication'
           ::Fabricate.singleton_class.prepend(FabricationPatch) if
```

Comment:
We can move this line into `TestProf.require` block and avoid checking for the constant.